R/get.R

```diff
@@ -278,16 +278,22 @@ get_unapproved_use <- function(approved_packages, used_packages) {
 #'
 #' @param file File path of file being run
 #'
-#' @importFrom lintr lint
-#'
 #' @return results from `lintr::lint()`
 #'
 #' @noRd
 #'
 get_lint_results <- function(file) {
+
+   if (!requireNamespace("lintr", quietly = TRUE)) {
+      message(strwrap("Linting will not be included in the log. Install the
+         lintr package to use the log.rx.lint feature.",
+         prefix = " ", initial = ""))
+      return()
+   }
+
    # lint file if option is turned on
    if (!is.logical(getOption('log.rx.lint'))) {
```

Comment:
```suggestion
   if (!is.logical(getOption('log.rx.lint'))) {
```
Are you expecting the person to pass a viable lintr option? If so, maybe checking more explicitly rather than just if the option is not a boolean would be more reliable. something like is.list(getOption('log.rx.lint')) since it does get passed to lintr::lint as the linters?